README.md

```diff
@@ -157,3 +158,16 @@ The following sensors are supported. Note that weather stations will only report
 | pm_in_temp_aqin               | Indoor temperature                                                  | Yes                                                 | No                  | °F      |
 | co2_in_aqin                   | Indoor CO2 meter reading                                            | Yes                                                 | No                  | ppm     |
 | co2_in_24h_aqin               | Indoor CO2 meter reading 24 hour average                            | Yes                                                 | No                  | ppm     |
+
+## Calculated sensors
+
+The following sensors are calculated using values from the weather station. Note that these will only be sent if the underlying values used for the calculation are available.
+
+| Name              | Description                                                    | Ambient Weather | Weather Underground | Unit      |
+| ----------------- | -------------------------------------------------------------- | --------------- | ------------------- | --------- |
+| dewpoint          | Outdoor dewpoint temperature                                   | Yes             | No                  | °F        |
+| dewpoint1..10     | Outdoor dewpoint temperature for sensors 1 through 10          | Yes             |                     | feelsLike | Windchill, heat index, or temperature | Yes | Yes | F |
+| feelsLike1..10    | Windchill, heat index, or temperature for sensors 1 through 10 | Yes             | Yes                 | F         |
+| lastRain          | Last time hourlyRain was > 0                                   | Yes             | Yes                 | timestamp |
+| Yes               | °F                                                             |
```

Comment:
This line is out of place. I think it is actually the tail of the malformed `dewpoint1..10` line